### PR TITLE
Api Gateway deployments - create before destroy

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/terraform/checks/resource/aws/example_APIGatewayDeploymentCreateBeforeDestroy/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+resource "aws_api_gateway_deployment" "fail2" {
+  rest_api_id = "some rest api id"
+  stage_name  = "some name"
+  lifecycle {
+    create_before_destroy=false
+  }
+  tags {
+    project = "ProjectName"
+  }
+}
```
Comment (gruebel) [inline tests/terraform/checks/resource/aws/example_APIGatewayDeploymentCreateBeforeDestroy/main.tf:33]: one of those two should be without the lifecycle block 😄

**Review by gruebel — COMMENTED** [review]
other than that, looks good 👍

File: tests/terraform/checks/resource/aws/example_APIGatewayDeploymentCreateBeforeDestroy/main.tf (hunk 1 above)
Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/aws/example_APIGatewayDeploymentCreateBeforeDestroy/main.tf]: yeah must have missed the edit

**Review by gruebel — APPROVED** [review]
🏆

**Pull request merged**